### ISSUE TITLE
[MuseSounds Playback] Portamento Playback on Musescore 4 

### ISSUE THREAD

**Issue by CombinedAtom** [musescore/MuseScore#22187]
### Your idea

A musesounds playback for portamento option in the glissando pallete where strings sort of slide in their instrument. 

### Problem to be solved

Usually the strings do this sort of glissando where they slowly slide up to the last note seen in this video

https://youtu.be/kPZR_XGUWds?si=v3qNlSySRbdIWwlj&t=59

**Comment by erinic04** [comment]
This is already a thing...

**Comment by DaddyLudwig** [comment]
Yeah, it’s an option along with chromatic glissando in the properties panel.
